pkg/common/cns-lib/volume/manager.go

```diff
@@ -742,6 +744,16 @@ func (m *defaultManager) QueryVolume(ctx context.Context, queryFilter cnstypes.C
 			log.Errorf("ConnectCns failed with err: %+v", err)
 			return nil, err
 		}
+		isvSphere70U3orAbove, err := cnsvsphere.IsvSphereVersion70U3orAbove(ctx, m.virtualCenter.Client.ServiceContent.About)
```

Comment:
As per the discussion in the meeting, we will need to add a feature gate and also check the feature-gate after confirming the 7.0.3 vCenter version.